tests/supervisors/test_multiprocess.py

```diff
@@ -19,3 +23,14 @@ def test_multiprocess_run():
     supervisor = Multiprocess(config, target=run, sockets=[])
     supervisor.signal_handler(sig=signal.SIGINT, frame=None)
     supervisor.run()
+
+
+@pytest.mark.skipif(
+    sys.platform.startswith("win"), reason="Skipping os.kill on windows"
+)
+def test_multiprocess_run2():
```

Comment:
Maybe a more descriptive name for this test…?

```suggestion
def test_multiprocess_cleanup_children():
    """
    Verify that stopping the multiprocess supervisor
    cleanly shuts down all child subprocesses.
    """
```

Also, given that `run` is a no-op in this test module, does this actually test anything, given that the subprocess will exit immediately (so `.shutdown()` will basically have nothing to do)?

Reply:
yep removed it